locales/en.json

```diff
@@ -1583,34 +1573,30 @@
     "steps": {
       "1": {
         "title": "Teleport/bridging",
-        "subtitle": "this transaction will get all required funds for purchase on required chain"
+        "tooltip": "this transaction will get all required funds for purchase on required chain"
       },
       "2": {
         "title": "Checking funds",
-        "subtitle": "Verifying fund arrival on destination chain for seamless operations."
+        "tooltip": "Verifying fund arrival on destination chain for seamless operations.<br> this may take few seconds"
       },
       "buy": {
-        "header": "<strong>Buying</strong> NFT",
```

Comment:
**suggestion**: html tags in translation file?? maybe use component interpolation

Reply:
my train of thought was if we need transalations in other languages something like this 
![CleanShot 2023-11-15 at 15 56 05@2x](https://github.com/kodadot/nft-gallery/assets/44554284/6fdcea32-6ad4-4a33-961d-3b81cb31dd0e)

would not work, that's why I had that strong inside the translations file